syntax = "proto3";

package NAlice.NMegamind.NMit.NScenarioState;

message TTestState {
    string CheckPhrase = 1;
}
